eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Ratify.hs

```diff
@@ -135,6 +142,53 @@ accepted RatifyEnv {reStakePoolDistr = PoolDistr poolDistr} gas =
       HardForkInitiation {} -> 101 % 100
       _ -> spoThreshold
 
+dRepAccepted :: forall era. RatifyEnv era -> GovActionState era -> Ratio Word64 -> Bool
+dRepAccepted RatifyEnv {reDRepDistr} GovActionState {gasDRepVotes, gasAction} threshold =
+  case dRepAcceptedRatio reDRepDistr gasDRepVotes gasAction of
+    Nothing
+      | threshold == 0 -> True
+      | otherwise -> False
+    Just ratio -> ratio >= threshold
```

Comment:
I am not sure if returning Nothing and handling it specially makes it clearer or not. Instead of Nothing we could just return `0`, in which case this would just become
```suggestion
  dRepAcceptedRatio reDRepDistr gasDRepVotes gasAction >= threshold
```

Feel free to ignore this suggestion if you believe we should make this case explicit.

Reply:
I was a little hesitant because the ratio is undefined rather than zero, but it does simplify the logic and it's maybe easier to understand what's going on, so i changed it  to remove the intermediate `Maybe`